testnets/prysmdevnet/chain.json

```diff
@@ -115,6 +120,9 @@
         "address": "https://prysm-rpc.validatorvn.com/",
         "provider": "ValidatorVN"
       },
+      {
+        "address": "https://prysm-testnet-rpc.validator247.com/",
+        "provider": "Validator247"
```

Comment:
this addition is missing a closing bracket. Please add new additions to the end of their respective arrays.

Reply:
I added it according to the format, sorry for the mistake

Reply:
And Please add new additions to the end of their respective arrays.

Reply:
added rpc/api/grpc/seed/persistent_peers...